PingPayments.Mimic/Autogiro/Update/Payment/V1/UpdatePaymentOperation.cs

```diff
@@ -0,0 +1,29 @@
+﻿using PingPayments.Mimic.Autogiro.Update.Payment.V1;
+using PingPayments.Shared;
+using System.Net.Http;
+using System.Threading.Tasks;
+using static PingPayments.Shared.Enums.HttpRequestTypeEnum;
+using static System.Net.HttpStatusCode;
+
+namespace PingPayments.Mimic.Autogiro.Update.Mandate.V1
+{
+    public class UpdatePaymentOperation : OperationBase<UpdatePaymentRequest, EmptyResponse>
+    {
+        public UpdatePaymentOperation(HttpClient httpClient) : base(httpClient) { }
+        public override async Task<EmptyResponse> ExecuteRequest(UpdatePaymentRequest updatePaymentRequest) =>
```

Comment:
```suggestion
namespace PingPayments.Mimic.Autogiro.Update.Payment.V1
{
    public class UpdatePaymentOperation : OperationBase<UpdatePaymentRequest, EmptyResponse>
    {
        public UpdatePaymentOperation(HttpClient httpClient) : base(httpClient) { }
        public override async Task<EmptyResponse> ExecuteRequest(UpdatePaymentRequest updatePaymentRequest) =>
```

Borde vara payment här va?